content/100-getting-started/02-setup-prisma/100-start-from-scratch/120-mongodb/100-connect-your-database.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: 'Connect your database'
```

Comment:
```suggestion
title: 'Connect your database (MongoDB)'
```

I am not sure this is the correct solution - but right now we have far to many pages with the same title "Connect your database" (This is bad both for UX and SEO). Possibly we need to implement via the switcher that the choices are added to the page title like suggested above.